engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -254,6 +265,20 @@ def scc_check_subscription_expiration(headers, login, system_token, logger, mode
 
       SccProxy.activations_fail_state(scc_systems_activations, headers, product)
     end
+
+    def scc_upgrade(auth, product, system_login, mode, logger)
+      uri = URI.parse(SYSTEM_PRODUCTS_URL)
+      http = Net::HTTP.new(uri.host, uri.port)
+      http.use_ssl = true
+      scc_request = prepare_scc_update_request(uri.path, product, auth, mode)
+      response = http.request(scc_request)
+      unless response.code_type == Net::HTTPOK
```

Comment:
```suggestion
      unless response.code_type == Net::HTTPCreated
```

Reply:
I will change the http status code, as mentioned, I followed the swagger documentation
![image](https://github.com/user-attachments/assets/8dea0538-af15-4030-a9b9-0dc152fe529a)